src/views/Layout.jsx

```diff
@@ -25,12 +27,16 @@ export function Layout() {
 						<NavLink to="/" className="Nav-link">
 							Home
 						</NavLink>
```

Comment:
**thought:** Should we take this one step further and have the `Home` button only show when a `listToken` doesn't exist? Essentially the inverse of what you wrote for `List` and `Add Item` nav buttons? Currently a user can click Home but it doesn't actually do anything.

Reply:
Thank you for catching that! Just made that change - Now it'll look really smooth when we add the `log out` feature :)

Reply:
This looks great. Thank you!